docs/framework/tutorial/code/app/org.example.CreativeCommonsBlog.json

```diff
@@ -0,0 +1,119 @@
+                        "mkdir -p assets"
+                    ]
+                },
+                {
+                    "type" : "file",
+                    "path" : "app.gresource.xml"
+                },
+                {
+                    "type" : "file",
+                    "path" : "app.yaml"
+                },
+                {
+                    "type" : "file",
+                    "path" : "credits.json"
+                },
+                {
+                    "type" : "file",
+                    "path" : "database_manifest.py"
+                },
+                {
+                    "type" : "file",
+                    "path" : "meson.build"
+                },
+                {
+                    "type" : "file",
+                    "path" : "org.example.CreativeCommonsBlog.desktop"
+                },
+                {
+                    "type" : "file",
+                    "path" : "output.shard"
+                },
+                {
+                    "type" : "file",
+                    "path" : "overrides.scss"
+                },
+                {
+                    "type" : "file",
+                    "path" : "sets.json"
+                },
+                {
+                    "type" : "file",
+                    "path" : "assets/icon.svg",
+                    "dest-filename" : "assets/icon.svg"
+                },
+                {
+                    "type" : "file",
+                    "path" : "assets/icon-64.png",
+                    "dest-filename" : "assets/icon-64.png"
+                },
+                {
+                    "type" : "file",
+                    "path" : "assets/icon-128.png",
+                    "dest-filename" : "assets/icon-128.png"
+                },
+                {
+                    "type" : "file",
+                    "path" : "assets/titleImage",
+                    "dest-filename" : "assets/titleImage"
+                },
+                {
+                    "type" : "script",
+                    "dest-filename" : "org.example.CreativeCommonsBlog",
+                    "commands" : [
+                        "ekn-app-runner org.example.CreativeCommonsBlog /app/share/org.example.CreativeCommonsBlog/app.gresource $@"
+                    ]
+                }
+            ],
+            "build-commands" : [
+                "install -Dm755 -t /app/bin ../org.example.CreativeCommonsBlog",
+                "install -Dm644 -t /app/share/applications ../org.example.CreativeCommonsBlog.desktop",
+                "install -Dm644 ../assets/icon-64.png /app/share/icons/hicolor/64x64/apps/org.example.CreativeCommonsBlog.png",
+                "install -Dm644 ../assets/icon-128.png /app/share/icons/hicolor/128x128/apps/org.example.CreativeCommonsBlog.png",
+                "install -Dm644 ../assets/icon.svg /app/share/icons/hicolor/scalable/apps/org.example.CreativeCommonsBlog.svg"
+            ]
+        }
+    ],
+    "build-options" : {
```

Comment:
This key here seems to be not in the original?

Reply:
ohh man, its the version builder modifies, I was pretty sure I took the right file
Can I force push to master to fix this?